portfolio/src/main/webapp/index.html

```diff
@@ -117,7 +117,7 @@ <h3>What I'm Listening To</h3>
           <h3>TV Quotes</h3>
           <div id="button-section">
             <p>Click here to get a quote from some of Ryan's favorite TV shows:</p>
-            <button onclick="getQuoteFromServer()">here</button>
```

Comment:
Usually it's better practice to keep all js code inside the js file. You can add these handlers inside a call to window.addEventListener('load', func() { ... } );

Reply:
All of my js code is inside of the js file. Also, the code you commented on got removed. getQuoteFromServer() (which got replaced with loadComments()) calls a function in my js file. I'm not sure how window.addEventListener works, but it probably does essentially the same thing as calling these js functions, right?

Reply:
I did more research into event listeners, and I think I understand the basic idea. We learned from the tutorials to call the methods as I did above, but if you think changing it to use event listeners is worthwhile I am happy to do so.

Reply:
I will address these changes in a new branch.